lightwood/encoders/__init__.py

```diff
@@ -35,6 +36,7 @@ class Text:
     ShortTextEncoder = ShortTextEncoder
```

Comment:
Also here

Reply:
short text encoder stays; it's meant for very short-form text i thought

Reply:
or rather it's used in "SHORT_TEXT" column data type

Reply:
Ah, sorry, this is Github acting weird

Reply:
I meant removing DistilBert

Reply:
Or maybe I misread, idk